src/app/login-page/login-page.component.ts

```diff
@@ -16,13 +16,23 @@ export class LoginPageComponent {
 
   constructor(public authService: AuthService, public userService: UserService, private router: Router) { }
 
-  onLogin(): void {
+  async onLogin(): Promise<void> {
```

Comment:
why do we need it?

Reply:
fixed in next PR. skipped to avoid conflicts